### DISCUSSION TITLE
move location of delete button on mobile

### DISCUSSION THREAD

**Discussion by anotheraccount1** [immich-app/immich#12122] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

currently, in mobile, when a photo is opened, the bottom of the screen contains actions...
share, edit, archive, and delete

in modern android phones, the navigation buttons are "soft keys" that disappear after a moment when an app is fulscreen. To regain access to android navigation buttons, the user "swipes up" from the bottom, and the navigation bar reappears.

However, the "Swipe up" gesture can very easily trigger the "delete" button in the immicah app.

Multiple solutions are possible, moving the delete button, or app the action buttons to the top of the screen would solve this very easily.

The only downside is that performing an action to a photo would take two hands, or a stretch of one hand to reach the top action buttons. but this seems like a feature not a bug, because the user should have to take a reasonable amount of effort to trigger an action. compared to now, where unintended actions are easily triggered accidentally.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by radh21301** [comment]
I am on IOS, so there might be a difference there.

Doesn't the app give a pop up to confirm the delete operation (after you touch 'delete')? But I definitely see your point. Not sure how google photos does it.

**Reply by anotheraccount1** [reply]
there might have been an update since i posted this. I see the behavior now of a popup saying "allow immich to delete this photo?" with a preview of the photo, and deny and allow buttons.
perhaps that is enough to resolved this item.
Although it's still recommend moving seldom used buttons, like the delete button, away from the modern OS areas like the bottom android buttons.

**Reply by Acidiphilium** [reply]
Delete button in my opinion is not a seldom used button and by far. It's one of the most used button if not the most. The popup warning should be enough. At least on my  Oneplus 12, I never unintentionally pressed the delete button.
